tests/integration/test_xlsx_regression.py

```
@@ -0,0 +1,172 @@
+        complete = [f"Перевод {i}" for i, _ in enumerate(texts)]
+        return [] if kind == "empty" else complete[:-1]
+    output = tmp_path / f"{kind}.xlsx"
+    with pytest.raises(Exception):
+        make_translator(RecordingModel(response)).translate_document(str(FIXTURE), str(output), "xlsx")
+    assert not output.exists()
+
+
+def test_reordered_model_response_is_rejected(tmp_path):
+    model = RecordingModel(lambda texts: list(reversed([f"Уникальный перевод {i}" for i, _ in enumerate(texts)])))
+    output = tmp_path / "reordered.xlsx"
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)